Source/JavaScriptCore/wasm/WasmCalleeGroup.h

```diff
@@ -244,9 +244,14 @@ class CalleeGroup final : public ThreadSafeRefCounted<CalleeGroup> {
     // FIXME: We should probably find some way to prune dead entries periodically.
     UncheckedKeyHashMap<uint32_t, ThreadSafeWeakPtr<OMGOSREntryCallee>, DefaultHash<uint32_t>, WTF::UnsignedWithZeroKeyHashTraits<uint32_t>> m_osrEntryCallees;
 #endif
+
     // functionCodeIndex -> functionCodeIndex of internal functions that have direct JIT callsites to the lhs.
-    // Note, this can grow over time since OMG inlining can add to the set of callers.
-    FixedVector<FixedBitVector> m_callers;
+    // Note, this can grow over time since OMG inlining can add to the set of callers and we'll tranisition from
+    // a sparse adjacency matrix to a bit vector based one if that's more space efficient.
+    // FIXME: This should be a WTF class and we should use it in the JIT Plans.
+    using SparseCallers = HashSet<uint32_t, DefaultHash<uint32_t>, WTF::UnsignedWithZeroKeyHashTraits<uint32_t>>;
```

Comment:
Use UncheckedKeyHashMap until we know this is fine in JSC.

Reply:
This is a `HashSet` and there doesn't seem to be an `UncheckedKeyHashSet`

Reply:
Yeah, UncheckedKeyHashSet is yet to be added.